lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,307 @@
+""" Tests for the v1 program enrollment API Views """
+import json
+from uuid import uuid4
+import ddt
+from django.contrib.auth.models import Permission
+from django.core.cache import cache
+from lms.djangoapps.program_enrollments.api.v1.constants import CourseEnrollmentResponseStatuses as CourseStatuses
+from lms.djangoapps.program_enrollments.models import ProgramCourseEnrollment
+from lms.djangoapps.program_enrollments.tests.factories import ProgramEnrollmentFactory, ProgramCourseEnrollmentFactory
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.lib.api.tests.mixins import JwtMixin
+from openedx.core.djangoapps.catalog.tests.factories import (
+    CourseFactory,
+    OrganizationFactory as CatalogOrganizationFactory,
+    ProgramFactory,
+)
+from openedx.core.djangolib.testing.utils import CacheIsolationMixin
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+from openedx.core.djangoapps.catalog.cache import PROGRAM_CACHE_KEY_TPL
+from rest_framework.test import APITestCase
+from student.tests.factories import UserFactory, GroupFactory
+
+
+class RequestMixin(JwtMixin):
```

Comment:
It seems unnecessary validate JWT / setup specific user permissions here since this endpoint is just called by the registrar service.

Reply:
Yeah, I'm taking this out